### ISSUE TITLE
Gradient calculation question

### ISSUE THREAD

**Issue by kampelmuehler** [NVlabs/stylegan2-ada-pytorch#118]
What's the use of adding `real_logits * 0` here?

https://github.com/NVlabs/stylegan2-ada-pytorch/blob/d4b2afe9c27e3c305b721bc886d2cb5229458eba/training/loss.py#L131

**Comment by SuwoongHeo** [comment]
I found a wonderful answer found in below. 

https://github.com/rosinality/stylegan2-pytorch/issues/48

Because the `DistributedDataParallel` does not properly work, as indicated in the pytorch docs, the dummy loss is added to trigger allReduce. 

> This module doesn’t work with [torch.autograd.grad()](https://pytorch.org/docs/stable/generated/torch.autograd.grad.html#torch.autograd.grad) (i.e. it will only work if gradients are to be accumulated in .grad attributes of parameters).
